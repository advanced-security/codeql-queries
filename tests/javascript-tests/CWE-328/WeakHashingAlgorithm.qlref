CWE-328/WeakHashingAlgorithm.ql