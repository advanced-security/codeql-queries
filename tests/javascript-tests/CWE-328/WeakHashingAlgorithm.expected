| weakcrypto.ts:7:12:7:48 | crypto. ... e(data) | weak hashing algorithms |
| weakcrypto.ts:8:12:8:49 | crypto. ... e(data) | weak hashing algorithms |
| weakcrypto.ts:14:1:14:21 | hashsha ... e(data) | weak hashing algorithms |
